src/Components/Publishing/Sections/Sections.tsx

```diff
@@ -280,7 +280,7 @@ export class Sections extends Component<Props, State> {
 
         ad = (
           <AdWrapper mt={marginTop}>
-            <NewDisplayCanvas
+            <DisplayAd
               pt={is300x50AdUnit(adDimension) ? 2 : 4} // add 20px to mobile leaderboard ads until this component is converted to <DisplayAd />
```

Comment:
@ashleyjelks you mentioned that I should be able to remove this `pt=...` line once I converted to `DisplayAd`, but I am concerned with the visual results of that change.

The current ads in staging/production look like this - notice the top padding.

![image](https://user-images.githubusercontent.com/1627089/60617740-fbb4d480-9d99-11e9-813a-10eabd64ec70.png)

After removing the `pt=...` here, they looked like this - notice the smaller top padding.

![image](https://user-images.githubusercontent.com/1627089/60617768-10916800-9d9a-11e9-8e68-88615839c0bb.png)

In this component we're setting `pt` to either 2 or 4, and [in `DisplayAd` it's setting it to either 0 or 2](https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Display/DisplayAd.tsx#L59). I don't know if this means that what is in `DisplayAd` is incorrect, or if we still need to apply the extra padding here. I left it as the latter until I could get your advice.

Reply:
@ashleyjelks I removed the `pt` specified in Sections.tsx now that we're handling it properly in DisplayAd.tsx.